src/main/java/com/ericsson/ei/handlers/RMQHandler.java

```diff
@@ -146,21 +148,41 @@ public void close() {
     }
 
     @Bean
-    protected Queue queue() {
+    protected Queue externalQueue() {
         return new Queue(rmqProperties.getQueueName(), true);
     }
 
+    @Bean
+    protected Queue internalQueue() {
+        return new Queue(rmqProperties.getWaitlistQueueName(), true);
+    }
+
     @Bean
     protected TopicExchange exchange() {
         return new TopicExchange(rmqProperties.getExchangeName());
     }
 
     @Bean
-    protected Binding binding(Queue queue, TopicExchange exchange) {
-        return BindingBuilder.bind(queue).to(exchange).with(rmqProperties.getBindingKey());
+    protected Binding binding() {
+        return BindingBuilder.bind(internalQueue()).to(exchange()).with(RMQProperties.WAITLIST_BINDING_KEY);
+    }
+
+    @Bean
+    public List<Binding> bindings() {
+        String[] bingingKeysArray = splitBindingKeys(rmqProperties.getBindingKeys());
```

Comment:
Don't we need to update the documentation that its support several binding key in the binding.key property?
https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/configuration.md

Reply:
Done